### PR TITLE
Drop EL 6, Debian Stretch and Ubuntu Xenial support in the builder

### PR DESCRIPTION
**Pull request by pieterlexis**
### Short description
EL6 will be EOL in 11 months.

### TODO
- [x] announce our intentions
- [x] wait for branches that will keep EL6 support

### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [x] compiled this code
- [x] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

**Comment by Habbie** [comment]
> I approve of the -content- of this, but I've marked this 'changes requested' because it is not time yet.

It is now time, but I have not done a new review of the content.

**Comment by Habbie** [comment]
Also, there are conflicts.

**Comment by pieterlexis** [comment]
> Also, there are conflicts.

Not anymore

**Comment by Habbie** [comment]
Missed spot: `dockerfiles/Dockerfile.debbuild-prepare:RUN if grep 'VERSION="9 (stretch)"' /etc/os-release; then \`

File: builder-support/specs/pdns-recursor.spec [hunk 1]
```diff
@@ -6,62 +6,46 @@ Group: System Environment/Daemons
...
 
 %ifarch ppc64 ppc64le
 BuildRequires: libatomic
 %endif
 
-%if 0%{?rhel} >= 7
-BuildRequires: protobuf-compiler
-BuildRequires: protobuf-devel
+%if 0%{?rhel} == 7
+# No fstrm in EPEL 8 yet
```
Comment (Habbie) [inline builder-support/specs/pdns-recursor.spec:35]:
```
Source       : fstrm-0.5.0-1.el8.src.rpm
Repository   : epel
```

File: builder-support/specs/pdns.spec [hunk 2]
```diff
@@ -277,100 +243,71 @@ chown -R pdns:pdns /var/lib/powerdns || :
...
-if [ $1 -ge 1 ]; then
-  /sbin/service pdns condrestart >/dev/null 2>&1 || :
-fi
-%endif
 
 %files
-%doc COPYING
-%doc README
-%doc pdns/bind-dnssec.4.2.0_to_4.3.0_schema.sqlite3.sql
-%doc pdns/bind-dnssec.schema.sqlite3.sql
```
Comment (Habbie) [inline builder-support/specs/pdns.spec:310]: where did these .sql files go?

Comment (Habbie) [reply builder-support/specs/pdns.spec:310]: away. Away is where they went :)

**Comment by Habbie** [comment]
> Missed spot: `dockerfiles/Dockerfile.debbuild-prepare:RUN if grep 'VERSION="9 (stretch)"' /etc/os-release; then \`

fixed

**Comment by Habbie** [comment]
Rebased to fix conflicts.

**Review by Habbie — APPROVED** [review]

**Review by omoerbeek — APPROVED** [review]

**Pull request merged**